sys/net/network_layer/fib/fib.c

```diff
@@ -955,11 +941,6 @@ int fib_sr_entry_add(fib_table_t *table, fib_sr_t *fib_sr,
         return -EFAULT;
     }
 
-    if (fib_sr_check_lifetime(fib_sr, &now) == -ENOENT) {
```

Comment:
sorry I commented in the commit, I move it here:
> @OlegHahm I don't get why this is obsolete?
Lets say a routing protocol has an aged source route with an expired lifetime.
It doesn't check it, but tries to add an hop into it.
Why should we let it proceed here?